templates/compositions/ec-component-navigation/navigation-list.html.twig

```diff
@@ -0,0 +1,49 @@
+{#
```

Comment:
I think we should change the name of this composition. It is not the navigation-list composition as defined in the ECL but rather our own implementation. Why not call like using the Drupal naming convention: local-tasks?

Reply:
Fair enough.